### PR TITLE
OCPBUGS-2524 correcting version numbers and adding variable substitution

### PR DESCRIPTION
**Pull request by kquinn1204**
[OCPBUGS-2524]:  Updating product version in appropriate places

Version(s):
  * PR applies to: 4.11, 4.12 and main
 
Issue:
https://issues.redhat.com/browse/OCPBUGS-2524 and https://issues.redhat.com/browse/OCPBUGS-2483

Link to docs preview:
https://52144--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-creating-nrop-cr_numa-aware

https://52144--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling.html#installing-the-numa-resources-operator_numa-aware

QE review:
- [x] QE has approved this change.

NOTE:
SME and QE approved a simple update of version numbers replacing with variable {product-version}

### CODE REVIEW

**Comment by kquinn1204** [comment]
@fromanirh @shajmakh can you please add LGTM for this PR. Using variable substitution, Branch Build will be replaced with whatever version the release is targeted for.

File: modules/cnf-installing-numa-resources-operator-cli.adoc [hunk 1]
```diff
@@ -94,8 +94,8 @@ $ oc get csv -n openshift-numaresources
 +
 .Example output
 
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
 NAME                            DISPLAY                      VERSION   REPLACES   PHASE
-numaresources-operator.v4.10.0  NUMA Resources Operator      4.10.0               Succeeded
+numaresources-operator.v{product-version}.2  NUMA Resources Operator      4.{product-version}.2               Succeeded
```
Comment (shajmakh) [inline modules/cnf-installing-numa-resources-operator-cli.adoc]: To clarify, you added the zstream (.2)just to resemble a real build right?

**Comment by shajmakh** [comment]
LGTM
thanks for the modifications!

File: modules/cnf-installing-numa-resources-operator-cli.adoc (hunk 1 above)
Comment (kquinn1204) [reply modules/cnf-installing-numa-resources-operator-cli.adoc]: Yes I ran this against a real build. Though i think variable messes up final output in docs.openshift.com

**Review by ffromani — COMMENTED** [review]
LGTM

**Review by kelbrown20 — COMMENTED** [review]
Looks great! 
/lgtm

**Pull request merged**